modules/470-chrony/images/chrony/start.sh

```diff
@@ -34,10 +34,13 @@ cmdallow 127/8
 allow 127/8
 bindaddress 127.0.0.1
 driftfile /var/run/chrony/chrony.drift
-makestep 1.0 3
+makestep 1.0 -1
 rtcsync
 EOF
 for NTP_SERVER in ${NTP_SERVERS}; do
+  if [ ${NTP_SERVER:0-1} != "." ]; then
+    NTP_SERVER=$NTP_SERVER"."
+  fi
```

Comment:
Please use trimSuffix func in helm for it.

Reply:
But we need to add a dot here as a suffix, not to remove it.

Reply:
This script runs on container startup, so we can't use helm functions here. Also, I'd added check for IP address in ntp servers.